### PR TITLE
`ci.yml`

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-cirq==0.13.0.dev20210901195302
-qiskit==0.28.0
-scipy==1.6.1
+cirq!=0.13.0.dev20210901195302
```
Comment (dowusu-antwi) [inline requirements.txt:1]: Shouldn't this be:
```suggestion
cirq~=0.13.0.dev20210901195302
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          if [ -f dev-requirements.txt ]; then pip install -r dev-requirements.txt; fi
+      - name: Coverage check
+        run: |
+          check/coverage
```
Comment (dowusu-antwi) [inline .github/workflows/ci.yml:129]: I remember this "no newline" being an issue in the past, but no tests fail so it seems fine.

**Review by dowusu-antwi — COMMENTED** [review]
Just a couple of things:

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vtomole) [reply .github/workflows/ci.yml]: Added a newline.

File: requirements.txt (hunk 1 above)
Comment (vtomole) [reply requirements.txt]: Ended up stealing the requirements from [cirq-superstaq](https://github.com/SupertechLabs/cirq-superstaq/pull/227)

**Review by dowusu-antwi — APPROVED** [review]
LGTM.

**Pull request merged**